packages/upload-api/src/blob/accept.js

```diff
@@ -50,7 +52,7 @@ export function blobAcceptProvider(context) {
         audience: DID.parse(space),
         with: context.id.toDIDKey(),
         nb: {
-          content: { digest: digest.bytes },
+          content: Link.create(rawCode, digest),
```

Comment:
Alan just did this change to use `digest`: https://github.com/w3s-project/w3up/pull/1480

This is supposed to use digest now, did you find any issue? Maybe you need to update dependencies

Reply:
one of the tests was failing 🤔 . I'll try to fix it

Reply:
alright this should be addressed